src/controller/CHIPDeviceController.cpp

```diff
@@ -1649,6 +1650,9 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
                                                   CommissioningDelegate * delegate, EndpointId endpoint,
                                                   Optional<System::Clock::Timeout> timeout)
 {
+    ChipLogProgress(Controller, "Performing next commissioning step '%s' with completion status = '%s'",
```

Comment:
There are similar prints to this in the PerformCommissioningStep function already. If you prefer to change the format, that's fine, but the duplicate prints in the called function should be removed because it's just double-printing the same information.

Reply:
As per discussion, removed the duplicated prints.